### DISCUSSION TITLE
Podman pull seems to consume more than 10GB

### DISCUSSION THREAD

**Discussion by zyrouge** [firefly-iii/firefly-iii#8778] [Firefly III] [answered]
I'm not sure why but pulling the image itself seems to occupy 10.5GB which is what is left on my VPS. I tried finding the `Dockerfile` of `fireflyiii:core`, but I couldn't find it. Is this the normal behaviour?

**Comment by JC5** [comment]
Not as far as I know, no.

https://github.com/firefly-iii/docker

**Comment by zyrouge** [comment]
I'm also not sure if `docker pull` has this same effect, I only use `podman` but seems like something is going wrong. I'm not sure how to find where it is going wrong since the only clue it says is "out of space when building", which I assume is that some image layer is taking too much of space. Will look into the respective `Dockerfile`s to find where it is happening.

**Comment by zyrouge** [comment, marked as answer]
Seems like some image related to MySQL seems to do this. I'm not a 100% sure, but this has happened with every image that has MySQL references.
